index.html

```diff
@@ -5,9 +5,11 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
+    <link rel="stylesheet" href="/css/">
```

Comment:
Why do you add css to this file? It shouldn't be there. We import styles in `main.js` and that's how we should keep it. Please delete that line.